app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -18,3 +18,37 @@
     padding-left: govuk-spacing(6);
   }
 }
+
+.gem-c-input--reveal-password {
+  // stylelint-disable declaration-block-no-duplicate-properties
+  display: table; // IE fallback
```

Comment:
Is this for IE <= 9? My instinct was to wonder whether adding extra code was worth this given it'd be mostly for IE8 users, but maybe I've misunderstood.

Reply:
This is actually for IE <= 11. Pretty much every version of IE has shonky support for flexbox. I experimented with the various hacks you can try to get around it but concluded the myriad of excess CSS wasn't worth the effort when it would fallback to using display table and display reasonably well anyway (see screenshots in description - only IE10 is noticeably a bit rubbish).

Reply:
I see, maybe my understanding is off but I thought IE 10 and IE 11 understood `display: flex` so this definition of `display: table` would be overwritten in IE > 9 and not have an effect?

Reply:
Correct. IE10 and 11 sort of understand `display: flex`, but not properly. I was getting weird layout results.